velox/docs/configs.rst

```diff
@@ -108,7 +108,13 @@ Expression Evaluation Configuration
    * - cast_to_int_by_truncate
      - bool
      - false
-     - This flags forces the cast from float/double to integer to be performed by truncating the decimal part instead of rounding.
+     - This flag forces the cast from float/double to integer to be performed by truncating the decimal part instead of rounding.
+   * - cast_string_to_date_iso_8601
+     - bool
+     - true
+     - This flag sets the cast from string to date accepts patterns with the "[+-](YYYY-MM-DD)" format (ISO 8601).
```

Comment:
```
If set, cast from string to date allows only ISO 8601 formatted strings: [+-](YYYY-MM-DD). Otherwise, allows all patterns supported by Spark. <TBD include a full list of pattern and a link to Spark's documentation if available>
```